docs/custom_v1.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Custom API Node.js
+---
+The Node.js OCR SDK supports [custom-built APIs](https://developers.mindee.com/docs/build-your-first-document-parsing-api). If your document isn't covered by one of Mindee's Off-the-Shelf APIs, you can create your own API using the[API Builder](https://platform.mindee.com/api-builder).
+
+# Quick-Start
+
+```js
+const mindee = require("mindee");
+// for TS or modules:
+// import * as mindee from "mindee";
+
+// Init a new client
+const mindeeClient = new mindee.Client({
+  apiKey: "my-api-key"
+});
+
+// Load a file from disk
+const inputSource = mindeeClient.docFromPath("/path/to/the/file.ext");
+
+// Create a custom endpoint for your product
+const customEndpoint = mindeeClient.createEndpoint(
+  "my-endpoint",
+  "my-account",
+  // "my-version" // Optional: set the version, defaults to "1"
+);
+
+// Parse it
+const apiResponse = mindeeClient
+  .parse(
+    mindee.product.CustomV1,
+    inputSource,
+    {
+      endpoint: customEndpoint,
+      cropper: true
+    }
+  );
+
+// Handle the response Promise
+apiResponse.then((resp) => {
+  // print a string summary
+  console.log(resp.document.toString());
+});
+```
+
+# Custom Endpoints
+
+You may have noticed in the previous step that in order to access a custom build, you will need to provide an account and an endpoint name at the very least. 
+
+
+Although it is optional, the version number should match the latest version of your build in most use-cases. If it is not set, it will default to "1".
+
```

Comment:
```suggestion

You may have noticed in the previous step that in order to access a custom build, you will need to provide an account and an endpoint name at the very least. 

Although it is optional, the version number should match the latest version of your build in most use-cases.
If it is not set, it will default to "1".

```